authors/apps/authentication/tests/test_registration.py

```diff
@@ -0,0 +1,54 @@
+from django.urls import reverse
+from rest_framework.test import APIClient, APITestCase
+from rest_framework.views import status
+from .test_data import (valid_user, empty_username,
+                        wrong_email2, short_password, missing_username_key)
+
+
+class UserLoginTest(APITestCase):
```

Comment:
Good work @Peace-Apple and @CryceTruly, I think you can also create a base class that can handle the client declarations eg the `setup()` also correct the naming of the file it says UserLoginTest yet tests are for registration.

Reply:
alright @marthamareal , thanks for the feedback, working on it

Reply:
Okay thanks, working on that